src/views/Home/components/Milestones/milestones.scss

```diff
@@ -1,3 +1,59 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/typography";
+@use "~stylesheets/responsiveness";
 
-/* Delete these lines when you begin development */
+.Milestones {
+  overflow: auto;
+  margin: 0;
+  min-height: 500px;
+  background: url("~images/components/milestones/background.png") repeat-y, #1a1542;
+  background-size: 100% auto;
+
+  &__title {
+    @include typography.title-text;
+
+    margin: var(--extra-large-spacer) 0;
+    text-shadow: 3px 3px #0085ff, -2px -2px rgba(255, 0, 0, 0.57);
+    text-align: center;
+  }
+
+  &__container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    max-width: 1500px;
+    margin: var(--medium-spacer) auto;
+  }
+
+  &__box {
+    margin: var(--medium-spacer);
+    min-width: 200px;
+    text-align: center;
+  }
+
+  &__box--top {
+    @include typography.title-text;
+
+    padding: 30px 45px;
+    background: rgb(96, 73, 155);
+    text-shadow: 3px 3px #0085ff, -2px -2px rgba(255, 0, 0, 0.57);
```

Comment:
I want to second Tim's comment above here as well. Although there isn't much repetition (just twice), we should put these in variables if we can. The hex & rgba values can go into a variable inside `variables.scss` and I could even go as far as saying that `3px 3px #0085ff, -2px -2px rgba(255, 0, 0, 0.57)` can be put into its own variable within this class as well!

Reply:
I agree that we should use variables for these! Any suggestion for naming? Some of the colors are only relevant to a few components w/ some variations.

Reply:
name suggestions for the color is really up to you. I really try and eyeballing it and giving it a name and just writing comments in the variables about where it's being used.